hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -471,7 +473,9 @@ public void validateContainerCommand(
     }
 
     State containerState = container.getContainerState();
-    if (!HddsUtils.isReadOnly(msg) && containerState != State.OPEN) {
+    if (!HddsUtils.isReadOnly(msg)
+        && (containerState != State.OPEN
+        && containerState != State.RECOVERING)) {
```

Comment:
Would it make sense to add a method for handling OPEN and RECOVERING state the same way in specific cases?  (I'm not sure what a good name for the method might be, though.)

Reply:
I tried to export into a method. Let's see if that's make sense.

Reply:
Tried to export into a method. Let's see if that make sense.